java-sdk-emitter/src/main/java/com/amazon/aocagent/MetricEmitter.java

```diff
@@ -14,12 +14,6 @@
  */
 package com.amazon.aocagent;
 
-import io.opentelemetry.OpenTelemetry;
-import io.opentelemetry.common.Labels;
-import io.opentelemetry.metrics.LongCounter;
-import io.opentelemetry.metrics.LongValueRecorder;
-import io.opentelemetry.metrics.Meter;
-
 public class MetricEmitter {
```

Comment:
For reference, ported this to new API here https://github.com/aws-observability/aws-otel-java-instrumentation/blob/main/sample-apps/springboot/src/main/java/com/amazon/sampleapp/MetricEmitter.java